LayoutTests/editing/input/ios/autocorrection-replaces-space-with-period.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html> <!-- webkit-test-runner [ useFlexibleViewport=true ] -->
```

Comment:
Is this test runner option necessary?

Reply:
Hm.. I think it's _generally_ a good idea for tests that exercise user interaction, to ensure that we don't end up trying to simulate user interaction at offscreen coordinates. That said, this test in particular is probably okay since the editor is at the top of the page.

Reply:
Oh that makes sense! Feel free to keep it around though, as best practice?